integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
```

Comment:
Crazy to see 0.10 is hanging on still, though I guess those stats are a year out of date now. Node 4 LTS support ends in April 2018: https://github.com/nodejs/LTS

As I understand it, adding `react-scripts` to `optionalDependencies` here was specifically to make create-react-app integration tests able to run on Node 4, if we wanted to. I don't think we care about going back that far for this particular integration tests. Let's chat more about the main library compatibility at some point.